deploy/scripts/src/mobile/utils/functions-mobile-journey.ts

```diff
@@ -125,181 +92,159 @@ function getBackendUrl (path: string, query?: Record<string, string>): string {
   return getUrl(path, env.backEndUrl, query)
 }
 
-export function doAuthorizeRequest (): void {
-  const res = postToVerifyURL()
-  isStatusCode201(res)
-  const verifyUrl = parseVerifyUrl(res)
-  const verifyRes = http.get(verifyUrl)
-  isStatusCode200(verifyRes)
-}
+export function checkAuthorizeRedirect (): void {
+  let verifyUrl: string
+  group('Post test client /start', () => {
+    const testClientRes = postTestClientStart()
+    isStatusCode201(testClientRes)
+    verifyUrl = parseVerifyUrl(testClientRes)
+  })
 
-export function startDcmawJourney (): void {
-  group('Start DCMAW Journey', () => {
-    const res = http.get(getFrontendUrl('/selectDevice'), {
-      tags: { name: 'Start DCMAW Journey' }
-    })
-    isStatusCode200(res)
-    isPageContentCorrect(res, 'Are you on a computer or a tablet right now?')
-    isPageRedirectCorrect(res, '/selectDevice')
+  group('Post /verifyAuthorizeRequest', () => {
+    const verifyRes = http.get(verifyUrl)
+    isStatusCode200(verifyRes)
+    isPageRedirectCorrect(verifyRes, '/selectDevice')
   })
 }
 
-export function checkSelectDeviceRedirect (device: DeviceType): void {
-  group(`Select device: ${device} from /selectdevice page`, () => {
-    const res = http.post(getFrontendUrl('/selectDevice'), { 'select-device-choice': device }, { tags: { name: 'Select Device Page' } })
+export function postSelectDeviceAndValidateRedirect (): void {
+  group('Post /selectDevice', () => {
```

Comment:
Is there a benefit of including details of request body in group name / tags, if we are only running a single scenario? (i.e. MAM iPhone Passport journey)

Reply:
Omitted as only the MAM iPhone passport journey is tested currently